ui/ducks/app/app.ts

```diff
@@ -409,10 +412,10 @@ export default function reduceApp(
         ...appState,
         sendInputCurrencySwitched: !appState.sendInputCurrencySwitched,
       };
-    case actionConstants.SET_NEW_CUSTOM_NETWORK_ADDED:
+    case actionConstants.SET_ADDED_NETWORK_CONFIGURATION_ID:
```

Comment:
What was this action for? It doesn't seem to be used

Reply:
Yep. Removed. What it was doing is now handled here: https://github.com/MetaMask/metamask-extension/pull/17421/files#diff-717d134b18ca11186a7f037ab64b913e703fcccf2f49f0487586c6f71ef8ab5fR345